README.md

```diff
@@ -1,12 +1,12 @@
 # [Kata Seeds](https://github.com/kata-seeds): Language (+ optionally a test framework)
-[![Build Status](https://travis-ci.org/kata-seeds/language-seed.svg?branch=master)](https://travis-ci.org/kata-seeds/language-seed)
+[![Build Status](https://travis-ci.org/kata-seeds/kata-seed-template.svg?branch=master)](https://travis-ci.org/kata-seeds/kata-seed-template)
```

Comment:
I left this as `language` so I could gsub it. Maybe we should standardize on LANGUAGE or something else super-obvious?